tests/tests_integration/test_gpu.py

```diff
@@ -1,24 +1,24 @@
-import re
+# import re
```

Comment:
Can you add a comment here regarding the reason for this, as you've mentioned in the comment in the PR?